src/components/clusterDetail/ClusterDetail.tsx

```diff
@@ -160,17 +163,14 @@ const ClusterDetail: React.FC<ClusterDetailProps> = ({
           Back to all clusters
         </ToolbarButton>
         <ToolbarSecondaryGroup>
-          {/* TODO(mlibra): Download cluster installation logs
           <ToolbarButton
             id="cluster-installation-logs-button"
             variant={ButtonVariant.link}
             style={{ textAlign: 'right' }}
```

Comment:
Nit: this style is apparently obsolete. Removing it does not affect anything. Same for the other button in the SecondaryGroup